src/integration/common/a11y.js

```diff
@@ -1,10 +1,9 @@
-export default ({ skipToContentText, headlineText }) => {
+export default ({ skipToContentText }) => {
   [amp, canonical].forEach((page) => {
     describe(`And using ${page.platform}`, () => {
       it('I can see a skip to content link that links to the main content of the page', () => {
         const skipToContentEl = page.getByText(skipToContentText);
-        const mainContentEl = page.getByText(headlineText);
-
+        const mainContentEl = page.document.querySelector('h1');
         expect(skipToContentEl.getAttribute('href')).toBe('#content');
         expect(mainContentEl.getAttribute('id')).toBe('content');
         expect(mainContentEl.getAttribute('tabindex')).toBe('-1');
```

Comment:
This variable is referring to `mainContent` but the query is selecting the h1 - it would be better to rename the variable for clarity.

```suggestion
        const h1El = page.document.querySelector('h1');
        expect(skipToContentEl.getAttribute('href')).toBe('#content');
        expect(h1El.getAttribute('id')).toBe('content');
        expect(h1El.getAttribute('tabindex')).toBe('-1');
```

Reply:
yeah agree ...this is a common test so for frontpages it would refre to main content and other pages the h1.

Reply:
May be should name it which would be common for content and h1.